core/src/main/scala/caliban/introspection/adt/__Directive.scala

```diff
@@ -17,4 +17,7 @@ case class __Directive(
       isRepeatable,
       locations.map(_.toDirectiveLocation)
     )
+
+  @deprecated("Use isRepeatable")
+  def repeatable: Boolean = isRepeatable
```

Comment:
Can we delete these then?

Reply:
Done 👍 My bad, had to head out in a hurry earlier and didn't cleanup properly